packages/edit-post/src/components/editor-regions/style.scss

```diff
@@ -1,47 +1,74 @@
+// We have to hide admin-bar in editor on mobile, in order to make the Mobile Safari experience bearable.
+// These two rules need to be moved to somewhere appropriate, and the !importants removed.
+#wpadminbar {
+	display: none;
+	@include break-small() {
+		display: initial;
+	}
+}
+#wpbody {
+	padding-top: 0 !important;
+	@include break-small() {
+		padding-top: initial !important;
+	}
+}
+
 .edit-post-editor-regions {
 	display: flex;
 	flex-direction: column;
 	height: 100%;
 	max-height: 100%;
-	position: relative;
 
-	// On Mobile keep a margin for the header and admin header
-	// as both of these are fixed
+	// Fill the entire screen on Mobile.
+	// We don't show the WP-adminbar, as editing on Mobile Safari is barely usable with it present.
+	position: fixed;
 	top: 0;
-	@include break-medium() {
-		margin-top: 0;
+	left: 0;
+	right: 0;
+	bottom: 0;
 
-		// On Desktop position the container as fixed to avoid scroll bleed.
-		position: fixed;
-		top: $admin-bar-height;
-		left: 0;
-		right: 0;
-		bottom: 0;
+	// Cancel the fixed positioning used on Mobile.
+	@include break-small() {
 		height: auto;
+		top: $admin-bar-height-big;
+	}
+
+	// Adjust to admin-bar going small.
+	@include break-medium() {
+		top: $admin-bar-height;
+
 		.is-fullscreen-mode & {
 			top: 0;
 		}
 	}
 }
+
 @include editor-left(".edit-post-editor-regions");
 
 .edit-post-editor-regions__body {
 	flex-grow: 1;
 	display: flex;
 
-	// On Mobile the header is fixed to keep HTML as scrollable.
-	@include break-medium() {
-		overflow: auto;
-	}
+	// Even on Mobile, we choose to scroll this element on its own.
+	// This helps enable a fixed-to-top toolbar that makes the editing experience
+	// on Mobile Safari usable.
+	// Unfortunately an issue still exists where if you swipe the top toolbar
+	// or beyond the bottom of the page when the soft keyboard is showing, you scroll
+	// the body element and can scroll the toolbar out of view.
+	// This is still preferable, though, as it allows the editor to function at all.
+	overflow: auto;
+
+	// In future versions of Mobile Safari, hopefully overscroll-behavior will be supported.
+	// This allows us to disallow the scroll-chaining and rubber-banding that is currently
+	// is the cause of the issue outlined above.
+	// In other words, the following behavior doesn't yet work in Safari, but if/when
+	// it is added, it should take care of the issue.
+	// See also: https://drafts.csswg.org/css-overscroll/
+	overscroll-behavior-y: none;
 }
 
 .edit-post-editor-regions__content {
 	flex-grow: 1;
-
-	// On Mobile the header is fixed to keep HTML as scrollable.
-	@include break-medium() {
-		overflow: auto;
```

Comment:
Is this change needed? I mean in desktop the content area should still be scrollable right? (I didn't test yet)

Reply:
I think this probably broke the sidebar scrolling. It is scrolling with the main area now, it shouldn't.